ci/postsubmit/push-to-staging/cloudbuild.yaml

```diff
@@ -0,0 +1,22 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 1200s
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+# Start by just pushing the image
+- name: 'gcr.io/k8s-testimages/bazelbuild:v20190916-ec59af8-0.29.1'
+  entrypoint: make
+  env:
+  - VERSION=$_GIT_TAG
```

Comment:
Looking at the `kops-controller-push` make target, does `KOPS_CONTROLLER_TAG` need to be defined? 

https://github.com/kubernetes/kops/blob/bb36e261e200b76d254f8fa9fb7964fe150b9052/Makefile#L916